MMI menu added

amigo421 opened the pull request.

Commit:
MMI menu added

aospan commented:
![mmi](https://user-images.githubusercontent.com/8099578/32693146-8cb9997c-c6f3-11e7-9861-d1adda2547d5.png)

strings incorrect position (see screenshot) when resize window to full screen.

aospan commented:
and two cosmetic suggestions:
 1) send string when "enter" key pressed
 2) remove popup help messages ("Enter CAM module value" and "Send CAM module value")

aospan committed:
fixes:
 - removed tips from MMI command and button
 - added Enter key action for MMI command

Commit:
fix for the layout of MMI form

aospan commented:
now it looks great ! thanks !
